src/Fields/field_tuples.jl

```diff
@@ -203,42 +203,6 @@ TracerFields(::Union{Tuple{}, Nothing}, grid, bcs) = NamedTuple()
 "Shortcut constructor for empty tracer fields."
 TracerFields(::NamedTuple{(), Tuple{}}, grid, bcs) = NamedTuple()
 
-#####
-##### Pressure fields tuples
-#####
```

Comment:
do we even use PressureFields anywhere now? should we delete in general? I can't see this being used anywhere but it's still being exported from Fields module and there are another 1-2 methods here and there...

Reply:
It was supposed to be deleted. Is this the only stray comment?

Reply:
Can you make these changes please instead of just requesting changes

Reply:
I was mostly asking for confirmation. I’ll deal with them don’t worry. 😉

Reply:
These lines are comments but there are PressureField related methods (not comments) elsewhere.

Reply:
Great. Done it!